irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/contract/service/EdcContractDefinitionServiceTest.java

```diff
@@ -226,4 +226,28 @@ void testCreateContractDefinitionResponse() throws IOException, JSONException {
         //THEN
         JSONAssert.assertEquals(jsonResponse, objectMapper.writeValueAsString(responseObj), false);
     }
+
+    @Test
+    void testEdcContractDefinitionQuerySpec() throws IOException, JSONException {
```

Comment:
What is the purpose of this test? Wouldn't it make more sense to build the object via constructor or builder and then compare it the the serialized JSON or vice-versa?

Reply:
This test verifies the structure of the class EdcContractDefinitionQuerySpec. 
I guess both approaches do the trick here and end in the same result